src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -747,6 +759,90 @@ void WindowImplWin32::processEvent(UINT message, WPARAM wParam, LPARAM lParam)
             break;
         }
 
+        // tell OS whether we're happy with the proposed changes to the window size
+        case WM_SIZING:
+        {
+            RECT* ncsRect = reinterpret_cast<RECT*>(lParam);
+            assert(ncsRect->left < ncsRect->right);
+            assert(ncsRect->top < ncsRect->bottom);
+            auto width  = static_cast<unsigned int>(ncsRect->right - ncsRect->left);
+            auto height = static_cast<unsigned int>(ncsRect->bottom - ncsRect->top);
+            bool isTop  = false; // are we being resized on the top? If not, it's the bottom
```

Comment:
```suggestion
            bool resizingFromTop  = false; // Are we being resized on the top? If not, it's the bottom
```
If this big comment is required to explain what the variable does, perhaps its name isn't expressive enough.